src/main/java/unicash/logic/parser/ListCommandParser.java

```diff
@@ -16,7 +16,7 @@ public class ListCommandParser implements Parser<ListCommand> {
     @Override
     public ListCommand parse(String userInput) throws ParseException {
         assert userInput != null : "userInput cannot be null";
-        //list command must not have any additional arguments
+        // list command must not have any additional arguments
         if (!userInput.trim().isBlank()) {
             throw new ParseException(ListCommand.MESSAGE_FAILURE);
         }
```

Comment:
I think this part was done by @sp4ce-cowboy , I was just adding a comment. I think we could further discuss this and if we were to revert this, it could be in a separate PR.

Reply:
Sounds good